pkg/model/serializer.go

```diff
@@ -724,10 +724,27 @@ func checkFileAndNotDirectory(path string) error {
 func (d *Dev) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	type devType Dev // Prevent recursion
 	dev := devType(*d)
-	err := unmarshal(&dev)
+	type hybridModeInfo struct {
```

Comment:
we could make this struct public and use in inside `Dev` struct to avoid duplicate fields (in `Dev` and in `hybridModeInfo`). 
````go
type HybridModeInfo struct {
  Workdir string `json:"workdir,omitempty" yaml:"workdir,omitempty"`
  Selector Selector `json:"selector,omitempty" yaml:"selector,omitempty"`
  . . .
}

type Dev struct {
. . .
HybridModeInfo
}
````
that means modify the use of the all fields related to HybridMode. (i.e Dev{HybridModeInfo.Selector: ...}. WDYT? @jLopezbarb

Reply:
I'd have tehm separate. They can evolve different and it doesn't make sense that the hybrid mode is inside the `Dev` since it's not a composition.